File: 251001/Stalbou/publisher/src/main/java/org/ex/distributed_computing/dto/request/NewsRequestDTO.java
package org.ex.distributed_computing.dto.request;

import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;

public record NewsRequestDTO(

    Long id,

    @NotNull
    Long authorId,

    @NotNull
    @Size(min = 2, max = 64)
    String title,

    @NotNull
    @Size(min = 4, max = 2048)
    String content
) {

}

File: 251004/Ubozhenko/DistComp_lab_1/DistComp_1/DTO/RequestDTO/NewsRequestDTO.cs
namespace DistComp_1.DTO.RequestDTO;

public class NewsRequestDTO
{
    public long Id { get; set; }
    public string Title { get; set; }
    
    public long AuthorId { get; set; }

    public string Content { get; set; }
    public DateTime Created { get; set; }
    public DateTime Modified { get; set; }

}